core/src/main/scala/scalaz/std/Future.scala

```diff
@@ -5,7 +5,8 @@ import scala.concurrent.{Await, CanAwait, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
 trait FutureInstances1 {
-  implicit def futureInstance(implicit ec: ExecutionContext): Monad[Future] with Cobind[Future] =
+
+  implicit def futureInstance(implicit ec: ExecutionContext): Cobind[Future] with MonadError[({type λ[α,β] = Future[β]})#λ, Throwable] with Catchable[Future] =
```

Comment:
There are two kinds; **scalaz datatype** and **std datatypes**.

**scalaz datatypes**: We can define type class instances in companion object or its super **classes** (eg. `sealed abstract class KleisliInstances9`). We can add new methods in companion object or super classes with keep binary compatibility.

**std datatypes**: We can't define type class instances in companion object. We **should** define in **trait** (not class) due to scalaz has some import style
- _a la carte_ import `import scalaz.std.scalaFuture._`
- `import scalaz.std.AllInstances._`
- `import scalaz.Scalaz._`

We **can't** add new methods in trait with keep binary compatibility (scala 2.11 or earlier)


In other words, adding new **std datatype** instances is harder than **Scalaz datatype** in scalaz7 😔 


Here is a unfortunate way add new typeclass instance.
https://github.com/xuwei-k/scalaz/commit/9c246bd5cfeed81bb9bc6d0f1e586f321a946388